source/linux/editing_files/nano.rst

```diff
@@ -0,0 +1,17 @@
+Nano
```

Comment:
Nano is not installed on default on our targets. We should at least add a note about this.

Reply:
I've added a note about it.